cnxdb/config.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+__all__ = ('discover_settings')
+
+
+def discover_settings():
+    """Discover settings from environment variables
+
+    :return: dictionary of settings
+    :rtype: dict
+
+    .. seealso::
+
+       See also :ref:`configuration_chapter` for environment variables,
+       defaults and required settings.
+
+    """
+    common_url = os.environ.get('DB_URL', None)
+    super_url = os.environ.get('DB_SUPER_URL', common_url)
```

Comment:
I'm with you on this. If you recall, I pushed hard against this when Paul wanted to use env vars for all our configs (~4yrs ago). I'm coming around to them, because they are an accepted practice in the containerized world because it's one less mounted file needed to run the container. Ideally, folks try to target being able to run a container without any additional configuration. I'm thinking it might have the added benefit of simplifying some config.

This also might begin to make more sense after you see the optional pyramid integration for this package.

Another reason for this is because I'm starting to think our configuration is getting pretty messy for a simple web app. I've got plans and some implementation that you'll be able to see in practice within Press sometime in the future.

Reply:
fair enough - we'll see.